### ISSUE TITLE
probit link function?

### ISSUE THREAD

**Issue by rsparapa** [jrdnmdhl/flexsurvcure#10]
Hi Jordan:

How hard would it be to add the probit link function?

Thanks,

Rodney

**Comment by jrdnmdhl** [comment]
Should be very easy. I believe it should only involve changing these lines to allow for additional cases. Basically, you are just defining the transform/inverse transform functions for the theta parameter based on the link argument.

https://github.com/jrdnmdhl/flexsurvcure/blob/ea31b1df9a9514bc169df1d45036350a77944c46/R/flexsurvcure.R#L110-L124

**Comment by rsparapa** [comment]
Hi Jordan:

Yes, that seems to do it.  So, i made the following change...
```
  } else if (link == "probit") {
        dist_list$transforms <- append(list(qnorm), dist_list$transforms)
        dist_list$inv.transforms <- append(list(pnorm), dist_list$inv.transforms)
  } else {
    stop("Link must be 'logistic', 'loglog', 'probit' or 'identity'")
  }
```
We tested this and it seems to work.  Can this come out in the next release?  We are 
trying to create a package that will depend on flexsurvcure.  So, please let me know.

Hope you are safe

Rodney

**Comment by jrdnmdhl** [comment]
This feature has been added to the master branch and will be released to CRAN as version 1.1.0 soon.

**Comment by jrdnmdhl** [comment]
It is now released: https://cran.r-project.org/web/packages/flexsurvcure/index.html
